reflections.md

```diff
@@ -0,0 +1,17 @@
+# Capstone Reflections
+With any large project that you've undertaken, especially one which requires a lot of personal growth and self-improvement, it's important to take time to reflect on the your process for completing the project and the outcomes of the decisions you made along the way.
+
+We're providing a set of suggested reflection prompts / questions that you can use to consider how things went with your capstone project, and we encourage you to add your own prompts as well!
+
+## Reflection prompts
+1. What went well during the capstone project?
```

Comment:
I think the context of this that we discussed was before the end of capstones and potentially during the third week